src/app/modules/addresses/addresses.component.html

```diff
@@ -0,0 +1,46 @@
+<main class="h-100" role="main">
+  <div class="container pt-4 pb-6">
+    <h1 class="mb-3" *ngIf="curentCustomerOrganization$ | async as curentCustomerOrganization">{{curentCustomerOrganization?.name}}</h1>
```

Comment:
You _probably_ need to move this if to the container/`div`-wrapper and don't repeat it below

Reply:
done